WalletWasabi/Tor/Control/TorControlClientFactory.cs

```diff
@@ -28,7 +28,7 @@ public class TorControlClientFactory
 
 	public TorControlClientFactory(IRandom? random = null)
 	{
-		Random = random ?? InsecureRandom.Instance;
+		Random = random ?? new UnsecureRandom();
```

Comment:
it was the easiest thing to do because `InsecureRandom` is now in WabiSabi package and it didn't sound good to tight Tor code to WabiSabi. Anyway, we can use it anyway, the only thing that we have to do is to replace the IRandom by a WasabiRandom and that should be pretty much all.